src/gtpu/encap.c

```diff
@@ -561,7 +561,7 @@ bool increment_and_check_counter(struct VolumeMeasurement *volmeasure, struct Vo
     return false;
 }
 
-int check_urr(struct pdr *pdr, struct far *far, u64 vol, u64 vol_mbqe, bool uplink) {
+int check_urr(struct pdr *pdr, struct far *far, u64 vol, u64 vol_mbqe, bool uplink, bool drop_pkt) {
```

Comment:
check_urr does not only perform checks but also send, I think we should rename this function